Extension/readme.developer.md

```diff
@@ -0,0 +1,297 @@
+# Developer Documentation
```

Comment:
Is there a way to add table of contents at the top of this document to see a summary of the contents/subheaders and for easy navigation?

Reply:
Uh ... turns out yes! I have the Markdown-All-In-One extension for vscode, and it's has that feature 😄